app/models/manageiq/providers/workflows/automation_manager/workflow_instance.rb

```diff
@@ -39,7 +39,15 @@ def queue_callback(state, message, _result)
 
   def run
     creds = credentials&.transform_values do |val|
-      ManageIQ::Password.try_decrypt(val)
+      if val.start_with?("$.")
+        name, field = val.match(/^\$\.(?<name>.+)\.(?<field>.+)$/).named_captures.values_at("name", "field")
+
+        # TODO: this isn't filtering out authentication records from other tenants
+        authentication = Rbac.filtered_object(ext_management_system.authentications.find_by!(:name => name), :userid => userid, :miq_group => miq_group, :class => Authentication)
```

Comment:
@kbrock I wonder if you could educate me on how `Rbac.filtered_object` works, this isn't filtering out auth records from other tenants

Reply:
I _think_ this is because there is no `Authentication` in `TENANT_ACCESS_STRATEGY` so we're returning everything.  If I add `'Authentication' => :ancestor_ids` then this works but I don't know if that will break other things

Reply:
It seems like dropping the `:class => Authentication` removed the need for `TENANT_ACCESS_STRATEGY` :shrug: